### PR TITLE
Coding Practice websites

### PR DESCRIPTION
**Pull request by rishesh007**
This is a document containing resources to start with coding and competitive programming.

### CODE REVIEW

**Comment by aditya-shirwatkar** [comment]
Why not rename the folder "programming-languages" to "programming" maybe and add this subsection of coding practices there 😅?
@rishesh007 @take2rohit

**Comment by take2rohit** [comment]
@aditya-shirwatkar @RajGhugare19 Put this file in a folder inside [software](https://github.com/IvLabs/resources/tree/master/software). Also, add it to the landing page readme

Update: I missed your comment @aditya-shirwatkar. So you are free to do so as per ur convenience!

**Comment by aditya-shirwatkar** [comment]
Okie will do that after merging with devel 👍

**Comment by rishesh007** [comment]
@aditya-shirwatkar don't merge for now as I am doing some changes. I will let us know when I am done with changes.

**Comment by take2rohit** [comment]
@aditya-shirwatkar @RajGhugare19 check for any errors. Properly arrange this in required folder and add in landing page readme.

File: coding-practices/README.md
Comment (aditya-shirwatkar) [inline coding-practices/README.md]: For convenience add _Table of Contents_

**Review by take2rohit — APPROVED** [review]

**Review by aditya-shirwatkar — APPROVED** [review]

**Pull request merged**
